docs/contribute/cumulative-docs/best-practices.md

```diff
@@ -0,0 +1,310 @@
+::::
+
+:::{warning}
+Do **not** use [inline annotations](/syntax/applies.md#inline-annotations) with headings, which will cause rendering issues in _On this page_.
+
+::::{image} ./images/heading-incorrect.png
+:screenshot:
+:alt: Rendering error when using inline applies_to with headings
+::::
+:::
+
+
+### Definition lists
+
+The recommended placement of `applies_to` badges in definition lists varies based what part(s) of the list item relate to the badge.
+
+#### If the badge is relevant to the entire contents of a list item, put it at the end of the term [definition-list-item-full]
+
+This means using an inline annotation at the end of the same line as the term. For example, on the Kibana [Advanced settings](https://www.elastic.co/docs/reference/kibana/advanced-settings#kibana-banners-settings) page, the entire `banners:linkColor` option is only available in Elastic Stack 9.1.0 and later.
+
+:::{image} ./images/definition-list-entire-item.png
+:screenshot:
+:alt: Correct use of applies_to with definition list item
+:::
+
+:::{warning}
+Do **not** put the `applies_to` badge at the beginning or end of the definition if it relates to the entire contents of the item.
+
+::::{image} ./images/definition-list-item-incorrect.png
+:screenshot:
+:alt: Incorrectly using inline applies_to with a definition list item
+::::
+:::
+
+#### If the badge is only relevant to a portion of the definition, follow the appropriate placement guidelines for the elements used in the definition [definition-list-item-part]
+
+This might include labeling just one of multiple paragraphs or one item in an ordered or unordered list. For example, on the ... page, ...
+
+::::{image} ./images/definition-list-portion-correct.png
+:screenshot:
+:alt: Correctly using inline applies_to in a portion of a definition list item
+::::
+
+
+### Ordered and unordered lists
+
+Reorganize content as needed so the `applies_to` badge is relevant to the entire contents of the list item.
+The recommended placement of the badge varies based on the purpose of the list.
+
+#### If the purpose of the list is to illustrate the difference between situations, put a badge at the start of each item [list-compare-applies_to]
+
+This could mean distinguishing between deployment types, products, lifecycles, or versions.
+Placing the badge at the beginning of the list item, allows the reader to scan the list for the item that is relevant to them.
+
+For example, the [Alerting and action settings in Kibana](https://www.elastic.co/docs/reference/kibana/configuration-reference/alerting-settings) page lists how the default value for the `xpack.actions.preconfigured.<connector-id>.config.defaultModel` setting varies in Serverless/Stack and across versions.
+
+::::{image} ./images/list-correct.png
+:screenshot:
+:alt:
+::::
+
+#### If the list just happens to have one or more items that are only relevant to a specific situation, put the badge at the end of the list item [list-other]
```

Comment:
I don't think I agree with this guidance. Curious to hear from others.

Reply:
I'm not sure about it either. This is just me attempting to summarize what I've seen across Slack, GitHub, etc over the last couple months. Either way we'll have some clean up if we want existing labels to conform to whichever guideline we decide on.

Reply:
More discussion in https://github.com/elastic/docs-builder/pull/1614#discussion_r2269234656. So far others have agreed we should be consistent for all list items.